### PR TITLE
Deleting database file while Deleting wallet

### PR DESCRIPTION
**Pull request by davidtavarez**

### CODE REVIEW

**Comment by DavidBurkett** [comment]
This file should already be deleted when the parent folder gets deleted

**Comment by davidtavarez** [comment]
> This file should already be deleted when the parent folder gets deleted

`RemoveFile` is supposed to delete the directory recursively because it is implementing `fs::remove_all` but actually is not deleting all the content neither the directory because the path is not empty.

File: src/Wallet/WalletDB/Sqlite/SqliteStore.cpp [hunk 1]
```diff
@@ -101,8 +101,15 @@ void SqliteStore::DeleteWallet(const std::string& username)
...
 	}
 
+	const fs::path dbFilePath = userDBPath / "wallet.db";
+	
+	if (!FileUtil::RemoveFile(dbFilePath)) {
+		WALLET_ERROR_F("Failed to delete wallet.db file for user: {}", username);
+		throw WALLET_STORE_EXCEPTION("Failed to delete wallet.");
+	}
+
 	if (!FileUtil::RemoveFile(userDBPath)) {
```
Comment (davidtavarez) [inline src/Wallet/WalletDB/Sqlite/SqliteStore.cpp:111]: This should delete the folder and the content inside the path but is not.

Comment (DavidBurkett) [reply src/Wallet/WalletDB/Sqlite/SqliteStore.cpp:111]: Because the file is locked. Make sure you don't have the file open

**Comment by davidtavarez** [comment]
Closing this PR since the proper solution is to unlock the database.

**Pull request closed without merging**